common/astyanax/src/main/java/com/bazaarvoice/emodb/common/cassandra/CqlCluster.java

```diff
@@ -63,6 +63,26 @@ private void registerMetrics() {
                 metrics.getOpenConnections());
 
         _metricRegistry.register(
+                MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "trashed-connections"),
+                metrics.getTrashedConnections());
+
+        _metricRegistry.register(
+                MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "executor-queue-depth"),
+                metrics.getExecutorQueueDepth());
+
+        _metricRegistry.register(
+                MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "blocking-executor-queue-depth"),
+                metrics.getBlockingExecutorQueueDepth());
+
+        _metricRegistry.register(
+                MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "reconnection-scheduler-task-count"),
+                metrics.getReconnectionSchedulerQueueSize());
+
+        _metricRegistry.register(
+                MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "task-scheduler-task-count"),
+                metrics.getTaskSchedulerQueueSize());
+
```

Comment:
Minor - metric naming may be changed from queue size to task count (for the two above) as it may make more sense. But the consistency and further search'bility may have more weight. I'm fine either way.